src/views/forms/dateQuestionnaireFields.ts

```diff
@@ -0,0 +1,27 @@
+export const fromDate = {
+  id: 'from-date',
+  name: 'from-date',
+  namePrefix: 'from-date',
+  fieldset: {
+    legend: {
+      text: 'You must enter a year, for example, 2007',
+    },
+  },
+  hint: {
+    text: 'You can choose to enter a day or month, for example, 27 3 2007',
+  },
+};
+
+export const toDate = {
```

Comment:
To generate this object structure, a utility function with open properties can be utilized. Invoking from the controller can be done by passing properties instead of exporting two objects with the same structure but different values.

eg:
```
interface IDateFieldAttributes {
  name: string;
  id: string;
  namePrefix: string;
  legendText?: string;
  hintText?: string;
}

const generateDateFieldAttribute = async (attributes: IDateFieldAttributes) => {
  return {};
};

export { generateDateFieldAttribute };
```

Reply:
yes